Kinvey/Kinvey/DataStore.swift

```diff
@@ -281,10 +304,14 @@ open class DataStore<T: Persistable> where T: NSObject {
     }
     
     /**
-     Gets a count of how many records that matches with the (optional) query passed by parameter.
-     - parameter query: The query used to filter the results
-     - parameter readPolicy: Enforces a different `ReadPolicy` otherwise use the client's `ReadPolicy`. Default value: `nil`
-     - parameter completionHandler: Completion handler to be called once the respose returns
+     Gets a count of how many records that matches with the (optional) query
+     passed by parameter.
+     - parameter query: (Optional) The query used to filter the results. Default
+     value: `nil`
+     - parameter readPolicy: (Optional) Enforces a different `ReadPolicy`
```

Comment:
This doesn't quite make sense. If you say "...otherwise uses the client's `ReadPolicy`, then the default is not `nil`. Actually, is it even the client's ReadPolicy? This seems more accurate - by default, the `ReadPolicy` is inferred from the store's type. If provided, the `ReadPolicy` overrides the default set by the store type.